config/_shared.json

```diff
@@ -18,6 +18,7 @@
 	"google_analytics_enabled": null,
 	"google_analytics_key": null,
 	"google_adwords_conversion_id": null,
+	"lucky_orange_enabled": null,
```

Comment:
they do feel like feature flags, but so do the google analytics flags as well. when I went through the config files and made the new linter, it felt like the top-level config keys are great places for actual config keys and settings while the feature flags are great for conditional execution: do this in production, disable everywhere else, enable in X Y Z, etc…

thanks for the ping @mcsf